app/assets/javascripts/components/navbar.jsx

```diff
@@ -0,0 +1,51 @@
+class Navbar extends React.Component {
+  constructor() {
+   super();
+  }
+
+  generate_tags(Tag, fields) {
+    const tags = fields.map((name, index) => {
+      return <Tag key={index} className={name}></Tag>;
```

Comment:
() should follow after return when we generate elements. For example `return (<Tag key={index} className={name}></Tag>);`